src/objects/target.gd

```diff
@@ -8,7 +8,6 @@ signal target_destroyed
 func on_raycast_hit(player_id: int) -> void:
 	if get_multiplayer().has_multiplayer_peer():
 		rpc("destroy_self", player_id)
-	destroy_self(player_id)
```

Comment:
This line can be removed since in the `@rpc` annotation below, it has "call_local" meaning it will automatically call it